src/services/OfferService.js

```diff
@@ -24,7 +24,7 @@ export default class OfferService {
   ): Promise<string> {
     const token = createToken()
 
-    const form = new forms[city](formData)
+    const form = new forms[city].Schema(formData)
```

Comment:
maybe we could transform it to 
```js
const { FormModel } = forms[city]
const form = new FormModel(formData)
```
for better readability. A `new` in front of an object looks odd to me :D